folium/features.py

```diff
@@ -588,12 +666,16 @@ def get_bounds(self):
                 ymax = none_max(y, ymax)
         return [
             [
-                self.data['transform']['translate'][1] + self.data['transform']['scale'][1] * ymin,  # noqa
-                self.data['transform']['translate'][0] + self.data['transform']['scale'][0] * xmin  # noqa
+                self.data['transform']['translate'][1] +
+                self.data['transform']['scale'][1] * ymin,  # noqa
+                self.data['transform']['translate'][0] +
+                self.data['transform']['scale'][0] * xmin  # noqa
```

Comment:
Hard to see if there is anything new here. If this is only line breaks please revert.